intdataplane/int_dataplane.go

```diff
@@ -108,6 +108,7 @@ type InternalDataplane struct {
 	ifaceMonitor     *ifacemonitor.InterfaceMonitor
 	ifaceUpdates     chan *ifaceUpdate
 	ifaceAddrUpdates chan *ifaceAddrsUpdate
+	ifaceRouteUpdates chan *string
```

Comment:
You should use a `chan string` instead of a `chan *string`; in go, strings are already pointer-like.